deploy/images/bicep/Dockerfile

```
@@ -0,0 +1,26 @@
+FROM alpine:latest AS base
+
+ARG TARGETARCH
+RUN if [ -z "${TARGETARCH}"]; then \
```

Sorry for this nit 👀 
```suggestion
RUN if [ -z "${TARGETARCH}" ]; then \
```